### PR TITLE
feat: implement get tokens list and delete token

### PR DESCRIPTION
**Pull request by hound672**
Hello!

On my backend I need to obtain tokens and delete one of them as well.
With thid PR I'd like to add these features.

Could you check it please?

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -0,0 +1,43 @@
+package main
```
Comment (hsluoyz) [inline main.go:1]: delete this file

**Review by hsluoyz — CHANGES_REQUESTED** [review]

**Comment by hsluoyz** [comment]
@hound672 fix 
image

**Comment by hound672** [comment]
done

**Comment by hound672** [comment]
Signed it

**Pull request merged**